### ISSUE TITLE
优化模型训练时loss的输出打印

### ISSUE THREAD

**Issue by NicoIer** [mindspore-ai/mindspore#229]
## Background
- 当使用MindSpore进行模型训练时,想要获取训练过程中的loss，以便于在后续绘图

## Introduction
- 在下面的代码中,LossMonitor只能打印,而没有办法获取训练中的所有loss值,希望有一个方法,可以获取训练中的所有loss值
```python
    model.train(num_epoch, ds_train, callbacks=[LossMonitor(per_print_times=ds_train.get_dataset_size() // 32)])
```

**Comment by cxl-ll** [comment]
Hello, welcome to MindSpore. You are advised to use the MindSIngsught tool to view the loss information.
https://www.mindspore.cn/mindinsight/docs/zh-CN/r2.2/summary_record.html
If you have any other questions, you are advised to raise the issue here.https://gitee.com/mindspore/mindspore/issues
